test/common/fit_amqp.py

```diff
@@ -0,0 +1,60 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+Author(s):
+Norton Luo
+
+'''
+import threading
+import pika
+import logging
+import fit_common
+
+
+class AMQP_worker(threading.Thread):
+    '''
+    This AMQP worker Class will creat another thread when initialized and runs asynchronously.
+    The externalcallback is the callback function entrance for user.
+    The callback function will be call when AMQP message is received.
+    Each test case can define its own callback and pass the function name to the AMQP class.
+    The timeout parameter specify how long the AMQP daemon will run. self.panic is called when timeout.
+    eg:
+    def callback(self, ch, method, properties, body):
+        print(" [x] %r:%r" % (method.routing_key, body))

```

Comment:
Logging output is shifting over to, well, logging. Please look in root/tests/stream-monitor/flogging/README.md for information on how to use it (still very rough documentation. The stories for templates/examples are in progress right now!). 

This, in particular, would be: logs.idl.debug("  [x]  %r:%r", method.routing_key, body)
(idl -> infrastructure.data logger)